homeassistant/components/sensor/synologydsm.py

```diff
@@ -74,6 +76,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+    vol.Optional(CONF_SSL, default=False): cv.boolean,
```

Comment:
Default True

Reply:
@pvizeli  the default port listed on the component is 5000 which is a not HTTPS. If you set the default to True, then we will need to change the default port to 5001 and that will be a breaking change. In this case, do you still want that set to True? I can make the changes if that is the case, but we will need to release a small note due to the breaking change.  

Please let me know your thoughts.

Reply:
Yeah, we do default the best possible solution and if a user don't want be save, he need that downgrade in config. In that case, set default port to 5001.

Reply:
Sounds good. I'll make sure to update the docs and update the PR soon.